Content.Server/Atmos/EntitySystems/AtmosphereSystem.HighPressureDelta.cs

```diff
@@ -201,37 +196,14 @@ private void ConsiderPressureDifference(GridAtmosphereComponent gridAtmosphere,
             if (!Resolve(uid, ref xform))
                 return;
 
-            // TODO ATMOS stuns?
-
-            var maxForce = MathF.Sqrt(pressureDifference) * 2.25f;
-            var moveProb = 100f;
-
-            if (component.PressureResistance > 0)
-                moveProb = MathF.Abs((pressureDifference / component.PressureResistance * MovedByPressureComponent.ProbabilityBasePercent) -
-                                     MovedByPressureComponent.ProbabilityOffset);
-
             // Can we yeet the thing (due to probability, strength, etc.)
-            if (moveProb > MovedByPressureComponent.ProbabilityOffset && _robustRandom.Prob(MathF.Min(moveProb / 100f, 1f))
-                                                                      && !float.IsPositiveInfinity(component.MoveResist)
-                                                                      && (physics.BodyType != BodyType.Static
-                                                                          && (maxForce >= (component.MoveResist * MovedByPressureComponent.MoveForcePushRatio)))
-                || (physics.BodyType == BodyType.Static && (maxForce >= (component.MoveResist * MovedByPressureComponent.MoveForceForcePushRatio))))
+            if (physics.BodyType != BodyType.Static && !float.IsPositiveInfinity(component.MoveResist))
             {
-                if (HasComp<MobStateComponent>(uid))
-                {
-                    AddMobMovedByPressure(uid, component, physics);
-                }
+                var moveForce = pressureDifference / physics.Mass;
 
-                if (maxForce > MovedByPressureComponent.ThrowForce)
+                if (moveForce > physics.Mass)
                 {
-                    var moveForce = maxForce;
-                    moveForce /= (throwTarget != EntityCoordinates.Invalid) ? SpaceWindPressureForceDivisorThrow : SpaceWindPressureForceDivisorPush;
-                    moveForce *= MathHelper.Clamp(moveProb, 0, 100);
-
-                    // Apply a sanity clamp to prevent being thrown through objects.
-                    var maxSafeForceForObject = SpaceWindMaxVelocity * physics.Mass;
```

Comment:
The old system uses SpaceWindMaxVelocity as a max, which seems to be getting ignored here.

Reply:
Eagle has volunteered to test this system for me today on his server, so I'm going to be getting some data from him with it in a live environment. In my own 6 hours of testing, I wasn't able to replicate Wizden's original issue of entities clipping into walls, at least under relatively tame atmospheric circumstances(Tame by my standards meant at most 5000kpa). That's not to say they won't clip if the force is sufficiently high. I'm also waiting for feedback from the original maintainer for this code on what the speed limit is. Which I should actually clamp down by the ApplyLinearImpulse.